### PR TITLE
Document the new block tools theme support and core block patterns

### PR DESCRIPTION
**Pull request by youknowriad**
Add the missing theme-support to the handbook

### CODE REVIEW

File: docs/designers-developers/developers/themes/theme-support.md [hunk 1]
```diff
@@ -268,6 +270,44 @@ add_theme_support( 'disable-custom-gradients' );
...
+add_theme_support( 'custom-line-height' );
+```
+
+### Supporting custom line heights
+
+Some blocks like paragraph and headings support customizing the line height. Themes can enable support for this feature with the following code:
+
+```php
+add_theme_support( 'custom-line-height' );
+```
```
Comment (ajitbohra) [inline docs/designers-developers/developers/themes/theme-support.md:287]: Duplicate

Comment (ajitbohra) [inline docs/designers-developers/developers/themes/theme-support.md:287]: Duplicate

Comment (ajitbohra) [inline docs/designers-developers/developers/themes/theme-support.md:287]: Duplicate

**Review by ajitbohra — CHANGES_REQUESTED** [review]

**Comment by youknowriad** [comment]
Good catch @ajitbohra it should be better now.

**Review by ajitbohra — APPROVED** [review]

**Comment by youknowriad** [comment]
thanks for the review @ajitbohra

**Pull request merged**